buildscripts/cspi-mgmt/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -ex
+
+rm /usr/local/bin/zrepl
+exec /usr/local/bin/cspi-mgmt start
```

Comment:
`start` argument is not needed, since code is not there to parse it in `cspi-mgmt`.
If you are planning to add, please add `version` command also.
Also move `openebs/maya/cmd/maya-apiserver/app/command/version.go` to `openebs/maya/pkg/version`, so that we don't need to have multiple copy of it in each `openebs/maya/cmd`.